src/umap.cpp

```diff
@@ -197,66 +173,14 @@ void umap_cfg_set_bufsize( int page_bufsize )
     umap_page_bufsize = page_bufsize;
 }
 
-_umap::_umap(  void* _mmap_addr, size_t _mmap_length, int _mmap_fd)
```

Comment:
I have more comments coming but will not be able to make them until either later today or tomrorow.  But, I wanted to pass these comments along to you for your consideration.

Thanks,